includes/pages/guest_login.php

```diff
@@ -179,7 +183,19 @@ function guest_register() {
       redirect('?');
     }
   }
-  
+  $password_required=true;
+  if (isset($_SESSION['ldap_user'])) {
+      $nick=$_SESSION['ldap_user'];
+      $ldaph=ldap_connect($ldap_host);
+      ldap_set_option($ldaph, LDAP_OPT_PROTOCOL_VERSION, 3);
+      $r = ldap_search($ldaph,"ou=users,".$ldap_basedn,"(&(objectClass=inetOrgPerson)(uid=".$nick."))",array("sn","givenName","mail"));
```

Comment:
hier

Reply:
Den "ou=users,"-Teil nicht hardcodieren, sondern auch in eine Variable analog zu $ldap_basedn packen, es gibt Leute, die nennen sowas auch "ou=people" oder man hat eine Unterteilung von Nutzern nach Abteilungen, ...

Bzgl. Escaping: http://php.net/manual/en/function.ldap-escape.php mit Option LDAP_ESCAPE_FILTER